sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/AdoDtoHelper.java

```diff
@@ -175,7 +175,7 @@ public boolean pushEntities(boolean onlyNewEntities)
 		throws DaoException, ServerConnectionException, ServerCommunicationException, NoConnectionException {
 		final AbstractAdoDao<ADO> dao = DatabaseHelper.getAdoDao(getAdoClass());
 
-		final List<ADO> modifiedAdos = onlyNewEntities ? dao.queryForNull(ADO.CHANGE_DATE) : dao.queryForEq(ADO.MODIFIED, true);
+		final List<ADO> modifiedAdos = onlyNewEntities ? dao.queryForEq(ADO.CHANGE_DATE, new Date(0)) : dao.queryForEq(ADO.MODIFIED, true);
```

Comment:
Move queryForEq... logic into the AbstractAdoDao class: queryForNew and queryForModified
Add unit test for those methods